tests/agent/base/scenario.rfs

```diff
@@ -0,0 +1,20 @@
+[Scenario]
+uploadmode = err
+scriptcount = 1
```

Comment:
If I remember correctly, If you wanted to run scenario[1] and scenario[2], you must set scriptcount=2

Reply:
yes correct, if the script count is 1 the manager will only load the first script